src/scss/variables/spacing.scss

```diff
@@ -8,3 +8,110 @@
     --kyn-spacing-gutter: 32px;
   }
 }
+
+//Spacer Definitions
+$spacer-unit-2: 2px;
+$spacer-unit-4: 4px;
+$spacer-unit-8: 8px;
+$spacer-unit-12: 12px;
+$spacer-unit-24: 24px;
+$spacer-unit-32: 32px;
+$spacer-unit-40: 40px;
+$spacer-unit-48: 48px;
+$spacer-unit-64: 64px;
+$spacer-unit-80: 80px;
+$spacer-unit-96: 96px;
+$spacer-unit-128: 128px;
+$spacer-unit-160: 160px;
```

Comment:
We can probably delete the SCSS variables, and just declare them as CSS vars like on line 47.